src/openassetio-python/cmodule/CMakeLists.txt

```diff
@@ -151,7 +181,7 @@ if (OPENASSETIO_ENABLE_PYTHON_STUBGEN)
             " pybind11-stubgen not found: status=${_pybind11_stubgen_exit_code}:"
             " ${_pybind11_stubgen_output}"
         )
-    endif()
+    endif ()
```

Comment:
Nit: an unexpected change, as in CMake, commands are _functions_ so having the parentheses bind to the command name without a space is what I'd expect to see.

Why did this change?

Reply:
:grin: This is CLion's auto-format of CMake files. I think most of our CMake conforms to CLion's preferences, but it's not been enforced.

Interesting point about commands being functions, I'd never thought about it that way, just assumed the brackets were an odd notational choice.

Reply:
So this comes down to different people with different editors applying different code preference rules?

Maybe something to stick on the backlog of 'these are the rules, make sure your editor conforms'.

Reply:
We have a related issue https://github.com/OpenAssetIO/OpenAssetIO/issues/1207 - I've added a comment there to ensure we look into styling some more.